xbmc/cores/VideoPlayer/VideoPlayer.h

```diff
@@ -262,6 +262,7 @@ class CVideoPlayer : public IPlayer, public CThread, public IVideoPlayer,
   float GetAVDelay() override;
   bool IsInMenu() const override;
   bool HasMenu() const override;
+  bool IsTopMenu;
```

Comment:
I think member variables should be named using a `m_` prefix, so in this case it would have to be `m_isTopMenu`. I'd probably also make this a bit more generic and use something like `m_isInOverlayMenu` or something like that. But I'm neither a core dev nor code authority.

Reply:
Ok I can change that. m_IsTopMenu makes more sense than overlay since it is not an overlay menu but the main menu of the movie. There are a problem a few more changes that need to be done (I expect) I'll wait other recommendations to fix everything at once.

Reply:
well, then `m_isMainMenu` maybe? Just not sure if everybody grasps what `top menu` means when he looks at the code without knowing the backgrounds.

Reply:
m_IsBlurayMainMenu ? Since this is a solution exclusive to blurays.